src/main/java/Model/Ticket.java

```diff
@@ -149,10 +160,25 @@ public LocalDateTime getCREATION_TIME() {
   }
 
 
+  public void share(String phone) {
+    System.out.println("Ticket is shared by phone: " + phone);
+  }
+
+  public void share(String phone, String email) {
+    System.out.println("Ticket is shared by phone: " + phone + ", and by email: " + email);
+  }
+
+
+  @Override
+  public void print() {
+    System.out.println("Ticket's print() method : " + this);
+  }
+
   @Override
   public String toString() {
```

Comment:
check the order of fields and methods

Reply:
Cheched the order of fields and methods.